ruby/conclusion/project_ruby_final.md

```diff
@@ -31,7 +32,7 @@ This is a great project to have as a part of your portfolio going forward becaus
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- [Illustrated rules of Chess](http://www.chessvariants.org/d.chess/chess.html)
-- [Chess notation for humans and computers](https://en.wikipedia.org/wiki/Chess_notation)
+- Chess Variant's [illustrated rules](http://www.chessvariants.org/d.chess/chess.html) of Chess.
+- Wikipedia's entry on [Chess notation for humans and computers](https://en.wikipedia.org/wiki/Chess_notation)
 - If you're feeling stuck, you should visit the community Discord!
-- Once you have completed the project, check out this [discussion](http://rubyquiz.com/quiz35.html) of the problem from RubyQuiz. **Warning**: this resource contains a blueprint for how to complete this project. That means, if you decide to peruse it, you won't get the opportunity to dig into this problem.
+- Once you have completed the project, check out the RubyQuiz [discussion](http://rubyquiz.com/quiz35.html) of the problem. **Warning**: this resource contains a blueprint for how to complete this project. That means, if you decide to peruse it, you won't get the opportunity to dig into this problem.
```

Comment:
The link text itself has to be descriptive enough on its own. even if you change the surrounding text to say `the RubyQuiz`, it's not part of the link text and navigating by link, a screenreader would just say "discussion - link", which isn't sufficiently descriptive of either where it leads or what it leads to.

```suggestion
- Once you have completed the project, check out this [discussion on writing chess in Ruby](http://rubyquiz.com/quiz35.html) from RubyQuiz. **Warning**: this resource contains a blueprint for how to complete this project. That means, if you decide to peruse it, you won't get the opportunity to dig into this problem.
```